tagger/utils.py

```diff
@@ -6,52 +6,48 @@
 from modules import shared, scripts
 from preload import default_ddp_path
 from tagger.preset import Preset
-from tagger.interrogator import Interrogator, DeepDanbooruInterrogator, WaifuDiffusionInterrogator
+from tagger.interrogator import Interrogator, DeepDanbooruInterrogator, WaifuDiffusionInterrogator, MLDanbooruInterrogator
 
 preset = Preset(Path(scripts.basedir(), 'presets'))
 
-interrogators: Dict[str, Interrogator] = {}
-
+interrogators: Dict[str, Interrogator] = {
+    'wd14-vit.v1': WaifuDiffusionInterrogator(
+        'WD14 ViT v1',
```

Comment:
If I understand correctly, this renaming is affecting the user directories, for those that already had the extension, this is not particularly nice, because of the duplicates